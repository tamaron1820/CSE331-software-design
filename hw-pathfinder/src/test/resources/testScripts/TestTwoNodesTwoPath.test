# Testing whether Dijkstra's algorithm works correctly
# if there are two nodes which have two edges

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2

AddEdge graph1 n1 n2 5.0
AddEdge graph1 n1 n2 10.0

FindPath graph1 n1 n2