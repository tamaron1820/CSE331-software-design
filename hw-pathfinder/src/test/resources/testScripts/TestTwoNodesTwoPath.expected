# Testing whether Dijkstra's algorithm works correctly
# if there are two nodes which have two edges

created graph graph1

added node n1 to graph1
added node n2 to graph1

added edge 5.000 from n1 to n2 in graph1
added edge 10.000 from n1 to n2 in graph1

path from n1 to n2:
n1 to n2 with weight 5.000
total cost: 5.000